meet-data/wpau/1303/entries.csv

```diff
@@ -0,0 +1,510 @@
+Place,Name,InternationalName,Country,BodyweightKg,Bench1Kg,Bench2Kg,Bench3Kg,TotalKg,WeightClassKg,Division,Sex,Event,Equipment,Tested,Squat1Kg,Squat2Kg,Squat3Kg,Deadlift1Kg,Deadlift2Kg,Deadlift3Kg,BestSquatKg,BestBenchKg,BestDeadliftKg,BirthYear
```

Comment:
The Country column here is incorrect -- those are City names. We can just delete that column.